src/player_hardcoded_effects.cpp

```diff
@@ -1590,6 +1590,7 @@ void Character::hardcoded_effects( effect &it )
                     } else if( limb == "hand" ) {
                         if( is_armed() && can_drop( weapon ).success() ) {
                             if( dice( 4, 4 ) > get_dex() ) {
+                                cancel_activity();  //Prevent segfaults from activities trying to access missing item
```

Comment:
Weapon?

Reply:
Whatever you are wielding is the characters weapon even if its a book or tool or whatever.

Reply:
I suppose it can technically be a melee weapon in any event...